tests/test_icgc.py

```diff
@@ -1,5 +1,6 @@
-from lib.cli import icgc_download_client
 from argparse import Namespace
+
+from icgcget import cli
```

Comment:
By source headers, do you just mean the copyright information, or do you want something else?